platform/src/apis/EducationBenchmarking.Platform.Api.Establishment/LocalAuthoritiesFunctions.cs

```diff
@@ -12,31 +12,31 @@ namespace EducationBenchmarking.Platform.Api.Establishment;
 public class LocalAuthoritiesFunctions
 {
     
-    [FunctionName(nameof(GetLocalAuthorityAsync))]
```

Comment:
I'm not a huge fan of this ReSharper recommendation to drop `Get` but I'll live with it elsewhere. On an `HttpTrigger` I really think it would be handy to keep them though to help distinguish between the HTTP method definition.

Reply:
I was surprised to see the warning. With the other `HttpTrigger` they are pre-fixed with the operation (e.g. query, search, create).

I agree it would be handy to have something to help distinguish - how about "SingleLocalAuthorityAsync"?